.appveyor.yml

```diff
@@ -35,8 +35,8 @@ environment:
       BUILD_STATIC : OFF
 
 
-# init:
-#   - ps: iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/appveyor/ci/master/scripts/enable-rdp.ps1'))
+init:
+  - ps: iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/appveyor/ci/master/scripts/enable-rdp.ps1'))
 
```

Comment:
why did you add these lines? I removed them in my merged PR

Reply:
These lines are those that allow to connect to appveyor remotelly. What I do usually is that they are there commented, and I do a commit uncommenting them. When I'm happy, I remove this commit from the history.